### DISCUSSION TITLE
Edit postinstall.in and include all the users of Mac machine be part of it

### DISCUSSION THREAD

**Discussion by jayjeetsinh** [lldpd/lldpd#494] [Q&A]
It appears that only the user who installs the app is the one be able to get the app functional. Other users who log in cannot make it work. if there a way to add a provision in the postinstall.in to have all the current and future members part of the group? LLDPD group so it works for everyone?

**Comment by vincentbernat** [comment]
`postinstall.in` only creates a system user for lldpd. It does not grant special access to the current user. What is not working? You should be able to use sudo to get lldpcli working.
